content/installation/java/package/scheduled-update-curl.md

```diff
@@ -0,0 +1,131 @@
+```
+
+* Create a script for installing the latest Java agent in the */etc/cron.daily* directory. Scripts in this directory execute once daily; as a result, the host updates the latest Java agent each day.
+
+* Use `tee` to create this script. Press `CTRL+D` when you've finished typing all the lines:
+
+```console
+$ sudo tee -a /etc/cron.daily/install-latest-contrast-agent > /dev/null
+#!/bin/bash -u
+
+CONTRAST_DIRECTORY=/opt/contrast
+CONTRAST_FILE_NAME=contrast-agent.jar
+
+curl --fail --silent --location "https://repository.sonatype.org/service/local/artifact/maven/redirect?r=central-proxy&g=com.contrastsecurity&a=contrast-agent&v=LATEST" -o /tmp/$CONTRAST_FILE_NAME
+if [ $? -ne 0 ]; then
+    echo "Failed to download Contrast Java agent" >&2
+    exit 1
+fi
+mv /tmp/$CONTRAST_FILE_NAME $CONTRAST_DIRECTORY/$CONTRAST_FILE_NAME
+```
+
+* Set the execute bit on the new script file:
+
+```console
+sudo chmod +x /etc/cron.daily/install-latest-contrast-agent
+```
+
+* To test the script, execute it and then verify that the file exists using `stat`:
+
+```console
+$ sudo /etc/cron.daily/install-latest-contrast-agent
+$ stat /opt/contrast/contrast-agent.jar
+stat /opt/contrast/contrast-agent.jar
+  File: /opt/contrast/contrast-agent.jar
+  Size: 10568283        Blocks: 20648      IO Block: 4096   regular file
+Device: 801h/2049d      Inode: 256034      Links: 1
+Access: (0644/-rw-r--r--)  Uid: (    0/    root)   Gid: (    0/    root)
+Access: 2019-04-11 02:02:01.265775928 +0000
+Modify: 2019-04-11 02:24:47.849796936 +0000
+Change: 2019-04-11 02:24:47.849796936 +0000
+ Birth: -
+```
+
+## Step 2: Gather Contrast account information 
+
+The Contrast agent requires some configuration to communicate with the Contrast UI. You'll need to obtain these [four properties](admin-orgsettings.html#apikey) from the Contrast UI:
+
+* The Contrast URL
+* Your Contrast API key
+* The agent username for your Contrast organization
+* The Service key for your Contrast organization's agent user
+
+The Contrast URL is *https://app.contrastsecurity.com/Contrast*, or the URL of your on-premises or private cloud instance. You can find the Contrast URL, your API key and agent username by going to the **user menu > Organization Settings > API tab** in the Contrast UI. You can find your Service key using the link in the **API** tab, or by navigating to the **user menu > Your Account > My Keys section**. 
+
+## Step 3: Configure Contrast
+
+When Contrast is installed on a Linux host, users typically want Contrast-enabled web applications on the host to share basic configuration parameters, such as the ones required to connect to Contrast UI. By convention, Contrast look for configuration in a YAML file at path */etc/contrast/java/contrast_securtiy.yaml* on Linux hosts.
+
+* Create the */etc/contrast/java* directory:
+
+```console
+sudo mkdir -p /etc/contrast/java
+```
+
+* Use `tee` to create the configuration file. Press `CTRL+D` when you've finished typing all the lines. 
+
```

Comment:
Re-commenting: 

@gilday Are they supposed to use CTRL+D before or after adding the info from the UI?

Reply:
After type all the info into the console, they press `CTRL+D` to write the information to the file and exist the `tee` program